tests/cephfs/bug-1980920.py

```diff
@@ -111,6 +111,9 @@ def run(ceph_cluster, **kw):
                 fs_util.remove_subvolume(
```

Comment:
Hi Julius,
As this is new sub volume it will not have any snapshots.
For the parent volume we are deleting the snapshot first and then removing the Sub volume.

Regards,
Amarnath